code/src/index.css

```diff
@@ -11,3 +28,188 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+ul {
+  list-style-type: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+input {
+  border: none;
+  font-size: 1rem;
+  background-color: transparent;
+  font-family: "Montserrat", sans-serif;
+}
+
+input:focus {
+  outline: none;
+}
+
+#root {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* flexbox */
+.container-column {
+  display: flex;
+  flex-direction: column;
+}
+.container-column-end {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+.container-space-between {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+.container-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.container-center-start {
+  display: flex;
+  align-items: center;
+}
+.container-center-end {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+.container-center-column {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.container-center-space-between {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+/* spacing */
+.p-3 {
+  padding: 0.75rem;
+}
+.m-1 {
+  margin: 0.25rem;
+}
+.my-1 {
+  margin: 0.25rem 0;
+}
```

Comment:
Now I understand your classname :) Smart to set standard spacing, did not think if that myself.